include/rocksdb/db.h

```diff
@@ -808,24 +808,36 @@ class DB {
   // stats should be included, or file stats approximation or both
   enum SizeApproximationFlags : uint8_t {
     NONE = 0,
-    INCLUDE_MEMTABLES = 1,
+    INCLUDE_MEMTABLES = 1 << 0,
     INCLUDE_FILES = 1 << 1
   };
 
+  struct SizeApproximationOptions {
+    // Defines whether the returned size should include the recently written
+    // data in the mem-tables (if the mem-table type supports it), data
+    // serialized to disk, or both.
+    SizeApproximationFlags include_flags = INCLUDE_FILES;
```

Comment:
It's annoying that we started with a bitmap. Now looks like what we should do is to have variable `bool include_memtables` and `bool include_files` in `SizeApproximationOptions`, other than this two levels of data structures. Can we do something like that?

Reply:
sure